introduction.md

```diff
@@ -7,3 +7,9 @@ Just introducing myself to the dojo. I am eager to learn about Github and how it
 # Favorite CodeSchool Path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?